ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -84,3 +82,45 @@ def test_funnel_trends_basic_post(self):
         self.assertEqual(len(response["result"]), 1)
         self.assertEqual(response["result"][0]["count"], 7)
         self.assertEqual(response["result"][0]["data"], [100, 100, 0, 0, 0, 0, 0])
+
+    def test_funnel_time_to_convert_auto_bins(self):
+        _create_person(distinct_ids=["user a"], team=self.team)
+        _create_person(distinct_ids=["user b"], team=self.team)
+        _create_person(distinct_ids=["user c"], team=self.team)
+
+        _create_event(event="step one", distinct_id="user a", team=self.team, timestamp="2021-06-08 18:00:00")
+        _create_event(event="step two", distinct_id="user a", team=self.team, timestamp="2021-06-08 19:00:00")
+        # Converted from 0 to 1 in 3600 s
+        _create_event(event="step three", distinct_id="user a", team=self.team, timestamp="2021-06-08 21:00:00")
+
+        _create_event(event="step one", distinct_id="user b", team=self.team, timestamp="2021-06-09 13:00:00")
+        _create_event(event="step two", distinct_id="user b", team=self.team, timestamp="2021-06-09 13:37:00")
+        # Converted from 0 to 1 in 2200 s
+
+        _create_event(event="step one", distinct_id="user c", team=self.team, timestamp="2021-06-11 07:00:00")
+        _create_event(event="step two", distinct_id="user c", team=self.team, timestamp="2021-06-12 06:00:00")
+        # Converted from 0 to 1 in 82_800 s
+
+        with self.settings(DEBUG=1):
```

Comment:
Why was `DEBUG=1` necessary here? O.o

Reply:
A ha. It was not after I quickly resolved a 500 using it. Removed.

Reply:
oooh, nice, does `DEBUG` print extra? (I usually go to the exception catcher and print there lol, this seems nicer)

Reply:
Yup, pretty good way to see the whole stack trace